src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
Does "Version" always have the build version number in it?  I thought that is what the "FileVersion" attribute was for?